src/nvim/quickfix.c

```diff
@@ -4399,7 +4403,7 @@ void ex_cexpr(exarg_T *eap)
   typval_T tv;
   if (eval0(eap->arg, &tv, NULL, true) != FAIL) {
     if ((tv.v_type == VAR_STRING && tv.vval.v_string != NULL)
-        || (tv.v_type == VAR_LIST && tv.vval.v_list != NULL)) {
+        || tv.v_type == VAR_LIST) {
```

Comment:
Does `eval0()` returns a list that is not `NULL`?

Reply:
? It could return either list, removal of this check makes it sure that NULL list is processed like empty one.